CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 🐛 *Bug Fixes*
 
 💅 *Improvements*
+* Added an informative error for situations where a mismatch in python versions between CE and the SDK cause deserialisation to fail.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏
```suggestion
* Added an informative error for situations where a mismatch in Python versions between CE and the SDK cause deserialisation to fail.
```